tools/topology/sof-tgl-max98373-rt5682.m4

```diff
@@ -92,12 +92,25 @@ dnl PIPELINE_PCM_ADD(pipeline,
 dnl     pipe id, pcm, max channels, format,
 dnl     frames, deadline, priority, core)
 
+ifelse(INCLUDE_IIR_EQ, `1',
+`
 # Low Latency playback pipeline 2 on PCM 1 using max 2 channels of s32le.
 # Schedule 48 frames per 1000us deadline on core 0 with priority 0
-PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+PIPELINE_PCM_ADD(sof/pipe-eq-iir-volume-playback.m4,
         2, 1, 2, s32le,
 	1000, 0, 0,
 	48000, 48000, 48000)
+'
+,
+`
+# Low Latency playback pipeline 2 on PCM 1 using max 2 channels of s32le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+        2, 1, 2, s32le,
+    1000, 0, 0,
+    48000, 48000, 48000)
```

Comment:
Please use tabs, not spaces.

Reply:
Fixed and tidied up the rest of the file as that had a mixture

Reply:
These things are not properly aligned, I think that's my main complaint here.